generation/apply_current_versions.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This script sets the "current-version" written in versions.txt applied to all
+# pom.xml files in this monorepo.
+# This script plays supplemental role just in case Release Please pull request
+# fails to update all files.
+
+# Usage:
+# # Run this script at the root of the monorepo
+# bash generation/apply_current_versions.sh
+
+function apply_module_versions_file() {
+  versions_file=$1
+
+  # Normally java-XXX. For the root pom.xml, it's "."
+  MODULE=$(dirname $versions_file)
+
+  for KV in $(cut -f1,3 -d: $versions_file |grep -v "#"); do
+    K=${KV%:*}; V=${KV#*:}
+    echo Key:$K, Value:$V;
+    pom_files="$(find $MODULE -maxdepth 3 -name pom.xml) pom.xml CoverageAggregator/pom.xml google-cloud-gapic-bom/pom.xml"
```

Comment:
Why do you have to list `CoverageAggregator` and `google-cloud-gapic-bom` explicitly? Wouldn't those poms be found automatically?

Reply:
It's finding pom.xml within the `$MODULE` directory. It does not try to find other 100 pom.xml files in other modules.

Reply:
Wouldn't it be more thorough, if it did? Just scan all pom.xml's for the version ref.

Reply:
It's unnecessarily slow. If you have a module that references a version in another module, then it makes sense. Do you know such module?

Reply:
It seems to be the case in at least `java-asset`.